dataman.tex

```diff
@@ -194,11 +194,14 @@ \section{Data Manager}\label{s:dataman-manager}
 module.exports = DataManager
 \end{minted}
 
-What our hubris made us miss in our first couple of attempts was that
-the \texttt{options} object controls how the parser behaves.
+The \texttt{papa.parse} function takes on two arguments: 
+the csv file to be parsed,
+and a callback.
```

Comment:
The second argument isn't a callback but an object in which callbacks can be provided. Nevertheless, your explanation is an improvement. I suggest something along the lines of

"... the CSV file to be parsed, and a configuration object that controls how the parser behaves. This configuration object is highly customisable. Here, we define `options` to instruct the parser to interpret ..."